core/utils/load_batch.py

```diff
@@ -89,27 +88,36 @@ def validate_header(sample_batch_data):
     error_cause
         If header was not validated, returns the error.
     """
-    invalid_col_name = []
-    missing_col_name = []
-    columns = list(sample_batch_data.columns.values)
-
-    invalid_col_name = [i for i in columns if i not in core.core_config.HEADING_BATCH]
-    project_id = sample_batch_data["sample_project"].unique().tolist()
-    projects_fields = get_sample_project_fields(project_id)
-    invalid_col_name = [i for i in invalid_col_name if i not in projects_fields]
-    missing_col_name = [i for i in projects_fields if i not in columns]
-
-    if invalid_col_name:
-        error_cause = core.core_config.ERROR_BATCH_INVALID_HEADER.copy()
-        error_cause.insert(1, ", ".join(invalid_col_name))
-        return " ".join(error_cause)
-    if missing_col_name:
-        error_cause = core.core_config.ERROR_BATCH_MISSING_HEADER.copy()
-        error_cause.insert(1, ", ".join(missing_col_name))
-        return " ".join(error_cause)
-
-
-def check_format_date(sample_batch_data):
+    fields = {}
+    heading_sample = []
+    for field in core.models.Samples._meta.get_fields():
```

Comment:
HEADING_BATCH contains the field_name. When displaying validation errors it shows the field_name and not the verbose_name, which makes a confusion because the user thinks that has to change verbose name to field_name